tests/avocado/test_ovmf_with_240_vcpus.py

```diff
@@ -0,0 +1,44 @@
+import os
+import shutil
+import sys
+
+from avocado import Test
+from base import VM
+
+
+class TestOvmfVcpus(Test):
+    """
+    :avocado: tags=ovmf
+    """
+
+    def setUp(self):
```

Comment:
Here we see a clear need for some sort of dependency support. This test has several environment dependencies and should be skipped if these dependencies are not met.

I wrote something about it in the avocado-devel discussion the other day, in the context of the thread "RFC: Guidelines for categorizing tests": http://www.mail-archive.com/avocado-devel@redhat.com/msg00643.html and http://www.mail-archive.com/avocado-devel@redhat.com/msg00645.html